lib/src/views/Calendar/DayWrapper.tsx

```diff
@@ -23,10 +23,9 @@ const DayWrapper: React.FC<DayWrapperProps> = ({
       role="presentation"
       onClick={dayInCurrentMonth && !disabled ? handleClick : undefined}
       onKeyPress={dayInCurrentMonth && !disabled ? handleClick : undefined}
+      children={children}
```

Comment:
For the convention in the React community and for people reading the source, I think that the previous approach would be less destabilizing.